code/modules/projectiles/guns/oddity_items.dm

```diff
@@ -68,4 +68,22 @@
 	damage_multiplier = 1.4
 	recoil_buildup = 4
 	one_hand_penalty = 6
-	penetration_multiplier = 1.1
\ No newline at end of file
+	penetration_multiplier = 1.1
+
+/obj/item/weapon/gun/projectile/boltgun/bluecross //Pending balance changes with
+	name = "\"Krag Jørgensen\" boltgun"
```

Comment:
You sure the norse `ø` character will show up on our current text formatting, or will it get replaced by a random cyrillic character like they do on OOC?

Reply:
Thats fine, name is likely going to change if it dosnt work